File: task01/c++/logger.cc
#include "logger.hh"

void Logger::print(std::string const &info, bool new_line) {
  std::cout << info;
  if (new_line)
    std::cout << std::endl;
}

void Logger::printStartInfo() {
  print("SE & IT-PM: Image generator -- C++ Version");
  print("==========================================");
}

File: task01/java/Logger.java
public class Logger {
  // --------------------------------------------------------------- //
  public static void print(String info, boolean new_line) {
    if (new_line) {
      System.out.println(info);
    } else {
      System.out.print(info);
    }
  }

  // --------------------------------------------------------------- //
  public static void print(String info) { print(info, true); }

  // --------------------------------------------------------------- //
  public static void printStartInfo() {
    print("SE & IT-PM: Image generator -- Java Version");
    print("===========================================");
  }
}